website/content/docs/commands/plugins/required.mdx

```diff
@@ -0,0 +1,27 @@
+---
+description: |
+  The "plugins required" command can install a plugin at a version constraint.
+page_title: plugins Command
+---
+
+# `plugins required`
+
+The `plugins required` command lists all plugins required by a Packer config and
+binaries.
```

Comment:
What do you mean by binaries?

Reply:
Good question, it's more: 
```suggestion
all the installed binaries that match the constraint. The first binary 
— also the most recent version — will be  the one picked by Packer in a build.
```

Reply:
I committed that one, LMK what you think ! :)